scene/resources/visual_shader.h

```diff
@@ -767,6 +767,27 @@ class VisualShaderNodeFrame : public VisualShaderNodeResizableBase {
 	VisualShaderNodeFrame();
 };
 
+// Deprecated, for compatibility only.
+class VisualShaderNodeComment : public VisualShaderNodeFrame {
+	GDCLASS(VisualShaderNodeComment, VisualShaderNodeFrame);
+
+	String description;
+
+protected:
+#ifndef DISABLE_DEPRECATED
+	static void _bind_compatibility_methods();
+	void set_description_compat_88014(const String &p_description);
```

Comment:
```suggestion
	void _set_description_bind_compat_88014(const String &p_description);
```
For both

Reply:
Oh sorry, I think you just mean the underscore.

Reply:
(it's also the style that'll be more concretely official if the docs get updated as currently proposed)

Reply:
Please do add the `bind` as well, it's preferred

Reply:
Alright, good to know, done :)